src/components/VitalStat.vue

```diff
@@ -0,0 +1,127 @@
+<script>
+/**
+ *   Displays a read-only stat
+ * @param stat {JSON} displayed object
+ * @DEPRECATED @param styleProp {String} style of displayed stat points
```

Comment:
ake je zmysel tohoto riadku? Navrhujem zmazat...

Reply:
...In current technical usage, for one to state that a feature is deprecated is merely a recommendation against using it. It is still possible to produce a program or product without heeding the deprecation...

...While a deprecated software feature remains in the software, its use may raise warning messages recommending alternative practices. Deprecated status may also indicate the feature will be removed in the future. Features are deprecated, rather than immediately removed, to provide backward compatibility and to give programmers time to bring affected code into compliance with the new standard....

https://en.wikipedia.org/wiki/Deprecation

Reply:
som to zmazal v tom svojom commitu, tak to tu zavrem